pySDC/implementations/problem_classes/Burgers.py

```diff
@@ -0,0 +1,291 @@
+import numpy as np
+
+from pySDC.implementations.datatype_classes.mesh import mesh, imex_mesh
+from pySDC.implementations.problem_classes.generic_spectral import GenericSpectralLinear
+
+
+class Burgers1D(GenericSpectralLinear):
+
+    dtype_u = mesh
+    dtype_f = imex_mesh
+
```

Comment:
Could you add some class documentation pls :wink:

Reply:
Make that ultra-documentation!